metricbeat/module/system/cpu/metrics/metrics_windows.go

```diff
@@ -0,0 +1,75 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/gosigar/sys/windows"
```

Comment:
You can probably switch over to https://github.com/elastic/go-windows. I think all the stuff from gosigar is there plus more.

Reply:
It looks like we're missing the `_NtQuerySystemInformation` call used to grab per-CPU stats.

Reply:
@fearful-symmetry as one of the reasons for these refactors is the deprecation of gosigar, would it make sense to add this missing syscall to go-windows and use it here? I am ok with this being done as a follow up.

It'd be nice to check if there is any other missing call and add it too.